redbot/core/commands/context.py

```diff
@@ -75,7 +75,9 @@ def __init__(self, **attrs):
             :func:`~redbot.core.utils.common_filters.filter_mass_mentions`.
             This must take a single `str` as an argument, and return
             the sanitized `str`.
-        \*\*kwargs
+        sanitize_roles : bool
```

Comment:
We should either include the default for this kwarg in the docstring or put the kwarg in func def so that sphinx puts it there automatically.